hosted/eks/p1/p1_suite_test.go

```diff
@@ -145,19 +146,308 @@ func syncK8sVersionUpgradeCheck(cluster *management.Cluster, client *rancher.Cli
 					}
 				}
 				return true
-			}, tools.SetTimeout(5*time.Minute), 10*time.Second).Should(BeTrue(), "Failed while waiting for nodegroup k8s upgrade to appear in EKSStatus.UpstreamSpec")
+			}, tools.SetTimeout(7*time.Minute), 10*time.Second).Should(BeTrue(), "Failed while waiting for nodegroup k8s upgrade to appear in EKSStatus.UpstreamSpec")
 
 			if !helpers.IsImport {
 				// For imported clusters, EKSConfig always has null values; so we check EKSConfig only when testing provisioned clusters
 				Expect(*cluster.EKSConfig.KubernetesVersion).To(Equal(upgradeToVersion))
 				for _, ng := range cluster.EKSConfig.NodeGroups {
+					Expect(ng.Version).ToNot(BeNil())
 					Expect(*ng.Version).To(BeEquivalentTo(upgradeToVersion), "EKSConfig.NodePools upgrade check failed")
 				}
 			}
 		})
 	}
 }
 
+func syncAWSToRancherCheck(cluster *management.Cluster, client *rancher.Client, k8sVersion, upgradeToVersion string) {
+	loggingTypes := []string{"api", "audit", "authenticator", "controllerManager", "scheduler"}
+	By("Enabling the LoggingTypes", func() {
+		err := helper.UpdateLoggingOnAWS(clusterName, region, loggingTypes, nil)
+		Expect(err).To(BeNil())
+
+		Eventually(func() bool {
+			cluster, err = client.Management.Cluster.ByID(cluster.ID)
+			Expect(err).To(BeNil())
+			updated := len(*cluster.EKSStatus.UpstreamSpec.LoggingTypes) == len(loggingTypes)
+			if !helpers.IsImport {
+				updated = updated && len(*cluster.EKSConfig.LoggingTypes) == len(loggingTypes)
+			}
+			return updated
+		}, "10m", "7s").Should(BeTrue(), "Timed out waiting for LoggingTypes update to appear in Rancher")
+
+		for _, lType := range loggingTypes {
```

Comment:
what about just `type`?

Reply:
`type` is a Go keyword, it cannot be used.